### PR TITLE
BZ-1172069 - Included success message notification

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/AbstractStartProcessFormDisplayer.java [hunk 1]
```diff
@@ -191,7 +193,9 @@ public void startProcess(Map<String, Object> params) {
             @Override
             public void callback(Long processInstanceId) {
                 newProcessInstanceEvent.fire(new NewProcessInstanceEvent(deploymentId, processInstanceId, processDefId, processName, 1));
-                jsniHelper.notifySuccessMessage(opener, "Process Id: " + processInstanceId + " started!");
+                final String message = "Process Id: " + processInstanceId + " started!";
```
Comment (manstis) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/main/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/AbstractStartProcessFormDisplayer.java]: Don't forget to i18n the message!

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/AbstractStartProcessFormDisplayerTest.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package org.jbpm.console.ng.pr.forms.client.display.displayers.process;
```
Comment (manstis) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/AbstractStartProcessFormDisplayerTest.java]: License header missing.

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/FTLStartProcessDisplayerImplTest.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package org.jbpm.console.ng.pr.forms.client.display.displayers.process;
```
Comment (manstis) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/FTLStartProcessDisplayerImplTest.java]: License header missing.

File: jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/PlaceManagerStartProcessDisplayerImplTest.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
+package org.jbpm.console.ng.pr.forms.client.display.displayers.process;
```
Comment (manstis) [inline jbpm-console-ng-process-runtime-forms/jbpm-console-ng-process-runtime-forms-client/src/test/java/org/jbpm/console/ng/pr/forms/client/display/displayers/process/PlaceManagerStartProcessDisplayerImplTest.java]: License header missing.

**Comment by cristianonicolai** [comment]
@manstis PR updated, please have a look when you get a chance, thanks.

**Comment by manstis** [comment]
+1 to merge.

**Comment by csadilek** [comment]
Thanks.

master: f0030605251bfc67a87e8357e6be0c161817e0e4

**Pull request closed without merging**